### PR TITLE
Missed text changes

### PR DESCRIPTION
**Pull request by stefan-oconnell**
### Description

List of proposed changes:

-
-

### CODE REVIEW

File: public/locales/fr/email.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "for-child-application": "<strong>Pour les demandes pour enfants,</strong> indiquez l'adresse courriel complète du parent ou du tuteur légal du requérant, telle qu'elle figure sur la demande de passeport."
   },
   "header-messages": {
-    "matches": "<strong>Assurez-vous que vos informations correspondent à votre preuve de citoyenneté et à votre pièce d'identité lors de votre demande de passeport.</strong> Votre preuve de citoyenneté pourrait être l'une des suivantes :",
+    "matches": "<strong>Assurez-vous que vos renseignements correspondent à votre preuve de citoyenneté et à votre pièce d'identité utilisèe lorsque vous avez présenté votre demande de passeport.</strong> Votre preuve de citoyenneté pourrait être l'une des suivantes :",
```
Comment (sebastien-comeau) [inline public/locales/fr/email.json:58]: `utilisèe` might be an error

Comment (stefan-oconnell) [reply public/locales/fr/email.json:58]: image

Should be intentional.

Comment (sebastien-comeau) [reply public/locales/fr/email.json:58]: `utilisèe` should be `utilisée`

Comment (stefan-oconnell) [reply public/locales/fr/email.json]: Resolved

**Review by sebastien-comeau — APPROVED** [review]

**Pull request merged**